src/sql/workbench/browser/modelComponents/dropdown.component.ts

```diff
@@ -109,6 +109,7 @@ export default class DropDownComponent extends ComponentBase implements ICompone
 
 		if (this.ariaLabel !== '') {
 			this._selectBox.setAriaLabel(this.ariaLabel);
+			this._editableDropdown.ariaLabel = this.ariaLabel;
```

Comment:
please make sure both dropdown and editable dropdown works properly

Reply:
Yes this makes editable dropdown have a name. Name for normal dropdown was added in #6775.

Reply:
my question was are both _selectbox and _editableDropdown available (not undefined) at the same time?

Reply:
Ah ok. Yes they are both available, not undefined.